backend/dataall/api/Objects/Environment/input_types.py

```diff
@@ -29,7 +29,6 @@
         gql.Argument('AwsAccountId', gql.NonNullableType(gql.String)),
         gql.Argument('region', gql.NonNullableType(gql.String)),
         gql.Argument('dashboardsEnabled', type=gql.Boolean),
-        gql.Argument('mlStudiosEnabled', type=gql.Boolean),
```

Comment:
Should be deleted from schema as well

Reply:
We should migrate `mlStudiosEnabled` to environment parameters. It's still used in `Environment` models, stacks and api.

Reply:
It is added in `backend/migrations/versions/5fc49baecea4_add_enviromental_parameters.py:76` I thought of having all environment parameter migrations together, but we can discuss this or maybe I am missing something else

Reply:
Never mind, I got what you were referring to!